meetings/zakim.md

```diff
@@ -0,0 +1,660 @@
+---
+title: The Zakim IRC Teleconference Agent
+toc: yes
+---
+
+## General Description {#general}
+
+The Zakim IRC "bot" is a Semantic Web agent ("swagent") that helps facilitate meetings using IRC.
+
+It is generally available on W3C's IRC server, under the "Zakim" nickname.
+
+In the following command descriptions, optional words in commands are indicated in square brackets. "`please`" can be abbreviated to "`pls`" if you wish to be polite yet still save keystrokes. ("`please`" is also accepted at the end of all commands.)
+
+## Bringing Zakim into your meeting channel {#invite}
+
+The command:
+
+`/invite Zakim <channel>`
+
+will bring Zakim on to your IRC channel.
+
+```irc
+[16:41] <Alan> /invite zakim &test
+[16:41] *** Zakim (zakim@irc.w3.org) has joined the channel
+```
+
+Some irc clients let you omit the channel name and will fill in the name of current channel. This is a feature of the irc client, not of Zakim.
+
+You might also want [RRSAgent](rrsagent.md) in your meeting channel to keep a record of the meeting in W3C Web space. To start logging the channel, invite the RRSAgent bot as follows:
+
+`/invite rrsagent <channel>`
+
+You can find the URL for the log being written by RRSAgent by typing the command:
+
+`rrsagent, bookmark`
+
+## Talking to Zakim {#addressing}
+
+Once Zakim is on your IRC channel, all commands to Zakim begin with
+
+`Zakim,`
+
+Zakim's commands are not case sensitive, but, in general, must be grammatically correct.
+
+Zakim will respond to normal irc lines and to `ACTION` lines (typically entered with the '`/me`' command in many irc clients). The '`/me`' form is especially recommended for avoiding clutter in [RRSAgent](rrsagent.md) logs; interactions that serve no useful purpose in the permanent irc record can be prefixed with '`/me`'. This typically includes '`q+`' and '`ack`' commands; e.g. '`q+`' will be logged by RRSAgent whereas '`/me q+`' will not (see [RRSAgent log recording](rrsagent.md#stop)). Zakim will generally mimic the same usage in the reply. For the commands such as '`agenda?`' to which Zakim will respond without being directly addressed, Zakim responds directly if he is addressed and in third-person ('`/me`') form if he was not addressed.
```

Comment:
```suggestion
Zakim will respond to normal irc lines and to `ACTION` lines (typically entered with the '`/me`' command in many irc clients). The '`/me`' form is especially recommended for avoiding clutter in [RRSAgent](rrsagent.md) logs; interactions that serve no useful purpose in the permanent irc record can be prefixed with '`/me`'. This typically includes '`q+`' and '`ack`' commands; e.g. '`q+`' will be logged by RRSAgent whereas '`/me q+`' will not (see [RRSAgent log recording](rrsagent.md#logoff)). Zakim will generally mimic the same usage in the reply. For the commands such as '`agenda?`' to which Zakim will respond without being directly addressed, Zakim responds directly if he is addressed and in third-person ('`/me`') form if he was not addressed.
```

Reply:
I'm also having a hard time parsing the last sentence of that paragraph...

Reply:
> I'm also having a hard time parsing the last sentence of that paragraph...

That describes how Zakim responds differently if you ask "Zakim, agenda?" or simply "agenda?". In that later case Zakim's reply will be off the record (i.e. the `/me` form).